examples/api/test/widgets/scroll_view/custom_scroll_view.1_test.dart

```diff
@@ -0,0 +1,244 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/scroll_view/custom_scroll_view.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Widget tree is visible', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.CustomScrollViewExampleApp());
+
+    // The crucial Widgets are:
+    // - Scaffold
+    //    - Appbar
+    //      - IconButton
+    //    - CustomScrollView
+    //       - SliverList (top, initially not existing)
+    //       - SliverList (bottom, with one element)
```

Comment:
Comments like this are nice, but they also carry a risk: if someone updates the test in the future but forgets to change the comments, things could get very confusing.

Since each `expect()` call already has a `reason`, I think it'd be best to remove the comments here.